include/seqan3/io/alignment_file/input.hpp

```diff
@@ -848,6 +848,9 @@ class alignment_file_input
     //!/brief Initialisation based on a filename.
     void init(std::filesystem::path & filename)
```

Comment:
> this should have a different name init_by_filename and the other one should be called init_by_format

yes that would be better. Naming is not my best skill :see_no_evil: 

> Why is this not directly done in the constructor?

Because I have 5 or 6 constructors that need 2 types of initialisation.

Reply:
But it was not renamed. I think it can be renamed in this PR. That is a small refactorisation that makes sense in this context.